server.py

```
@@ -757,6 +841,31 @@ def _main():
 
     fastapi_app = FastAPI(title=f"{agent_name()} — protoAgent")
 
+    # --- Scheduler lifecycle ------------------------------------------------
+    # The local scheduler needs an asyncio polling task; the Workstacean
+    # adapter is a no-op start/stop. Both implement the same contract so
+    # we just call through. on_event is preferred over a lifespan
+    # context manager here — the rest of the boot is sync (uvicorn.run
+    # is the only blocking call) and FastAPI fires startup/shutdown
+    # around it.
+    @fastapi_app.on_event("startup")
+    async def _scheduler_startup():
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_scheduler_startup`

Add return type annotation: `None`

(ANN202)